Gems/ROS2/Code/Source/RobotImporter/URDF/UrdfParser.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "UrdfParser.h"
+
+#include <fstream>
+
+#include <AzCore/Debug/Trace.h>
+#include <AzCore/std/string/string.h>
+
+#include <urdf_model/model.h>
+
+namespace ROS2
+{
+    namespace Internal
+    {
+        void checkIfCurrentLocaleHasDotAsADecimalSeparator()
```

Comment:
This function should be UpperCamelCase per the coding standard: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#function-names